tensorflow/lite/micro/examples/micro_speech/zephyr_riscv/CMakeLists.txt

```diff
@@ -0,0 +1,116 @@
+cmake_minimum_required(VERSION 3.13.1)
```

Comment:
I don't think we need to merge this CMake. We should rather stick to the convention and move changed TF files to the src directory inside the Zephyr's project directory.